kano_video/logic/youtube.py

```diff
@@ -5,16 +5,28 @@
 # Copyright (C) 2014-2015 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
 #
-# Interracts with the YouTube API to retreive video links
+# Interracts with the YouTube API v3.0 to retreive video links
 #
 
 import os
 from shutil import rmtree
 from kano.utils import requests_get_json, run_cmd
 from kano.logging import logger
+from apiclient.discovery import build
+from apiclient.errors import HttpError
+from oauth2client.tools import argparser
+from isodate import parse_duration # Used to process the ISO 8601 duation format used for the video length
+
+DEVELOPER_KEY = "" # Insert your public api key here from console.developers.google.com
+YOUTUBE_API_SERVICE_NAME = "youtube"
+YOUTUBE_API_VERSION = "v3" # As of 1/6/2015 v3.0 api is still an experimental API
+
 
 tmp_dir = '/tmp/kano-video'
 last_search_count = 0
+nextPageToken = ""
+prevPageToken = ""
+prevPageIndex = 1
```

Comment:
Can we stick to underscores in variable names rather than camel case? We try to stick to [PEP 8 naming guidelines](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions).